UKHomeOffice/sgmr-service
# SPCR-176 Upgrade to Webpack v5

## Description
Upgrade to Webpack 5

- remove file-loader as being deprecated
- update webpack.config to support removal of file-loader
- add css-loader and style-loader to support removal of file-loader
- move main.scss file to support new webpack build requirements
- update relative paths due to main.scss move

## To test

- remove `package-lock` and `node modules`
- run `npm i`
- run `npm build`
- > should build without errors (but will have warnings see below)
- run `npm build:dev`
- > should build without errors (but will have warnings see below)
- run `npm start`
- > should run without errors (but will have warnings see below)
- go to `localhost:8080`
- > styles should look normal

Expected warnings
- we expect multiple warnings about `math` in css, this is part of govuk frontend and known
- we expect a few warnings about file size, we have a ticket open to investigate this


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
